taxi_service/settings.py

```diff
@@ -143,3 +145,5 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+CRISPY_TEMPLATE_PACK = "bootstrap4"
```

Comment:
The CRISPY_TEMPLATE_PACK setting is defined twice in the file (lines 81 and 149). Consider removing one of these definitions to avoid redundancy.